common/src/Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet/src/Default/JsonSchemaDotNetSchemaValidator.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Microsoft.Azure.IIoT.Validators {
+    using Json.Schema;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+    using System.Text.Json;
+    
+    /// <summary>
+    /// JsonSchema.Net schema validator.
+    /// </summary>
+    public class JsonSchemaDotNetSchemaValidator : IJsonSchemaValidator {
```

Comment:
We do not use `DotNet` in the names of our projects or classes. Please remove this part from the class name.

Reply:
JsonSchemaDotNet (JsonSchema.net) is a name of nuget package which we use for schema validation hence it's reflected in the project and class name(s). This follows the existing naming convention of NewtonSoft nuget package where it's reflected in class names like NewtonSoftJsonSerializer.cs. Wonder if we still need to change the names?

Reply:
OK, I see the reasoning behind the naming here. I will mark this and other comment about `DotNet` as resolved.